src/devtools/client/webconsole/actions/filters.js

```diff
@@ -26,17 +40,17 @@ export function filterTextSet(text) {
 }
 
 export function filterToggle(filter) {
-  return ({ dispatch, getState, prefsService }) => {
+  return ({ dispatch, getState }) => {
     const filters = getAllFilters(getState());
-    const filtersState = { ...filters, [filter]: !filters[filter] };
+    const newValue = !filters[filter];
+    const filtersState = { ...filters, [filter]: newValue };
 
     dispatch({
       type: FILTER_TOGGLE,
       filtersState,
       filter,
     });
-
-    prefsService.setBoolPref(PREFS.FILTER[filter.toUpperCase()], filtersState[filter]);
```

Comment:
Once upon a time prefsService used to do stuff. Now it's toothless. We should clean this up in the next pass.

Reply:
https://github.com/RecordReplay/devtools/blame/fc08cf873c1844587d97895bf89174eca4bf511c/src/devtools/client/webconsole/utils/prefs.js